.husky/_/husky.sh

```
@@ -16,21 +1,9 @@
+echo "husky - DEPRECATED
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)